protected/humhub/modules/space/config.php

```diff
@@ -4,15 +4,16 @@
 use humhub\modules\user\models\User;
 use humhub\modules\space\Events;
 use humhub\modules\space\Module;
-use humhub\components\console\Application;
 use humhub\commands\IntegrityController;
+use humhub\widgets\TopMenu;
 
 return [
     'id' => 'space',
     'class' => Module::class,
     'isCoreModule' => true,
     'urlManagerRules' => [
-        ['class' => 'humhub\modules\space\components\UrlRule']
+        ['class' => 'humhub\modules\space\components\UrlRule'],
+        'spaces' => 'space/spaces',
```

Comment:
`/spaces`  should be good here

Reply:
Currently I can open Space Directory page by url `http://humhub.local/spaces`, but if I change to `'spaces' => '/spaces',` then it is impossible, because only url `http://humhub.local/space/spaces` works. As I understand this rule creates an alias `spaces` for the route `space/spaces`.

Reply:
@yurabakhtin Ah ok, sorry, it's great as it is!